### ISSUE TITLE
Table id not used as a default index

### ISSUE THREAD

**Issue by stevenmathews** [rethinkdb/horizon#773]
Server version: 2.0.0-beta-7
Client version: 2.0.0-beta-7

Used `hz schema apply` to set the indexes on my collections. With only `[collections.houses]` I got `Error: Collection "houses" has no index matching [["id"]].` Adding:

```
[[collections.houses.indexes]]
fields = [['id']]
```

fixed the issue. However the next collection:

```
[collections.tenants]
[[collections.tenants.indexes]]
fields = [['house']]
```

works without including `['id']`.

Not sure if this is intended but thought I should mention it.

**Comment by danielmewes** [comment]
Thanks for reporting this @stevenmathews .

I'd expect lookups by ID to always work.

Which Horizon operation exactly gives you the `has no index matching [["id"]]` error?

**Comment by stevenmathews** [comment]
`Collection.find`. I mentioned that it worked for the tenant collection above but was incorrect. Had just done `Collection.watch` without the `find`.

Trying `fields = [['id'], ['house']]` so that I could do `find` and `findAll` on the tenants collection caused neither of them to work. Same error as above.

**Comment by Tryneus** [comment]
I believe this is fixed in PR #778 - the primary index was being registered incorrectly, so it wasn't matching any requests.
